### PR TITLE
Clicking on a district will add it to the compare section with a limi…

### PR DESCRIPTION
**Pull request by Obleo33**
…t of 2 districts

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,6 +8,7 @@
     "react-scripts": "0.9.0"
   },
   "dependencies": {
+    "classnames": "^2.2.5",
```
Comment (Obleo33) [inline package.json:11]: I think we can use this to toggle classes

File: src/HeadCount/HeadCount.js [hunk 2]
```diff
@@ -2,6 +2,7 @@ import React, { Component } from 'react';
 import DistrictCards from '../DistrictCards/DistrictCards';
 import kinderData from '../../data/kindergartners_in_full_day_program.js';
 import DistrictRepository from '../helper';
+import CompareDistricts from '../CompareDistricts/CompareDistricts'
```
Comment (Obleo33) [inline src/HeadCount/HeadCount.js]: added to store compared districts

File: src/HeadCount/HeadCount.js [hunk 3]
```diff
@@ -10,7 +11,8 @@ export default class HeadCount extends Component {
     super()
     this.state = {
       data: {},
-      search: []
+      search: [],
+      compare: []
```
Comment (Obleo33) [inline src/HeadCount/HeadCount.js:15]: Added new state to store the compared districts.

File: src/HeadCount/HeadCount.js [hunk 4]
```diff
@@ -32,8 +34,12 @@ export default class HeadCount extends Component {
   }
 
   handleClick(location) {
-    console.log(location+ 'this was clicked');
-
+    if(this.state.compare.length<2){
```
Comment (Obleo33) [inline src/HeadCount/HeadCount.js:37]: push to array if there are less than 2 in it

File: src/HeadCount/HeadCount.js [hunk 5]
```diff
@@ -32,8 +34,12 @@ export default class HeadCount extends Component {
...
+      this.state.compare.push(this.state.data.findByName(location))
+    }
+    this.state.compare.shift()
```
Comment (Obleo33) [inline src/HeadCount/HeadCount.js:40]: shift first item out of array if you are trying to add and the array already has 2 in it.

Comment (becs919) [reply src/HeadCount/HeadCount.js:40]: yasssss

**Comment by becs919** [comment]
nice work!

**Pull request merged**